components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -414,21 +416,41 @@ export class Panel extends React.Component<Props, State> {
     return defaultContribution
   }
 
+  initiateDelayCounter = () => {
+    setTimeout(() => {
```

Comment:
this timeout needs to be cleared before set again. We also need to clear it on component unmount